cb-talent-development/employer-style-base
# EM-1359: Mobile Subnav (ESB)

**Purpose:**
> As a mobile user, I should have the same ability to see the page's sub-nav, but in a collapsed view, so I can easily jump to different parts of the subscription page.

This adds back a collapsed version of the subnav for our product pages. The initial link that appears when the menu is first seen changes based on the section you scroll by. The menu also updates itself based on the section you're on so there are no redundant links.

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/EM-1359

**Changes:**
* Changes to setup
  * Modified: _flyout.scss - added a new mixin for the collapsed subnav with responsive styles.

* Architectural changes
  * N/A

* Migrations
  * N/A
  
* Library changes
  * N/A

* Side effects
  * Primarily affects any page that uses `.product-subnav` as its class for the subnav. These pages are primarily the product pages. Also affects the small-biz page.

**Screenshots**
* Before
N/A
* After
<img width="626" alt="closed" src="https://user-images.githubusercontent.com/5348098/27409311-6c767986-56a6-11e7-8fe1-d784a2e42ab4.png">
<img width="627" alt="open" src="https://user-images.githubusercontent.com/5348098/27409312-6c867962-56a6-11e7-8a6d-e3547831ec32.png">

**QA Links:**
http://web.employer-5.development.c66.me/

**How to Verify These Changes**
* Specific pages to visit
  * All the product pages that display a subnavigation
  * http://web.employer-5.development.c66.me/recruiting-solutions/small-business-subscription-plans-and-pricing   - NOTE: Currently the switching functionality does not work on this page. This is not related to the subnav itself as switching does not happen at all regardless of screen size.

* Steps to take / responsive considerations
See the steps found in https://github.com/cbdr/employer/pull/890

**Relevant PRs/Dependencies:**
https://github.com/cbdr/employer/pull/890

**Additional Information**
